examples/typescript/README.md

```diff
@@ -0,0 +1,21 @@
+# Pass Go data to TypeScript
+
+## Tasks
+
+### generate
+
+```bash
+templ generate
+```
+
+### build-ts
+
+```bash
+esbuild --bundle --minify --outfile=assets/js/main.js ts/src/index.ts
```

Comment:
Did you run `go install github.com/evanw/esbuild/cmd/esbuild@latest` or `npx esbuild ...`?